coopersystem-fsd/pcc-shell
# Updated plugins

## Description

- Updated plugins used by the customer

## Steps to test

1. Go into root folder create .env file then type the command docker-compose up --build -d in terminal

**Expected behavior: Docker will created all the WordPress Structure and you can access the production environment
